packages/arb-util/common/timeticks.go

```diff
@@ -21,28 +21,18 @@ import (
 	"time"
 )
 
-const (
-	AverageSecondsPerBlock = 2
-)
+const TicksPerBlock = int64(1000)
 
 type TimeTicks struct {
 	Val *big.Int
 }
 
-var _timeConversionFactor *big.Int
-var _timeTicksPerSecond *big.Int
-
-func init() {
-	_timeTicksPerSecond = big.NewInt(1000)
-	_timeConversionFactor = new(big.Int).Mul(big.NewInt(13), _timeTicksPerSecond)
-}
-
 func TimeFromBlockNum(blockNum *TimeBlocks) TimeTicks {
-	return TimeTicks{new(big.Int).Mul(_timeConversionFactor, blockNum.AsInt())}
+	return TimeTicks{new(big.Int).Mul(big.NewInt(TicksPerBlock), blockNum.AsInt())}
 }
 
 func TimeFromSeconds(seconds int64) TimeTicks {
-	return TimeTicks{new(big.Int).Mul(_timeTicksPerSecond, big.NewInt(seconds))}
+	return TimeTicks{big.NewInt(int64(time.Duration(seconds*TicksPerBlock) * time.Second / _durationPerBlock))}
```

Comment:
These should probably be titles "Ticks" instead of "Time" for clarity

Reply:
Done.